Fix make repair failing on stale venv pip shebangs after workspace move

## Summary

`make repair` can fail to repair a moved/renamed workspace because it recreates `.venv` and then invokes a stale `pip` launcher with an old shebang.

## Reproduction

In this workspace on March 31, 2026:

```bash
make validate
make repair
```

Initial validation reported:

- `venv has stale shebangs (workspace was renamed/moved)`

`make repair` then failed during the setup-dev phase with:

```text
/bin/bash: line 1: /home/roland/daddy_camp/.venv/bin/pip: cannot execute: required file not found
make[1]: *** [Makefile:143: /home/roland/daddy_camp/.make/setup-dev.done] Error 127
make: *** [Makefile:101: repair] Error 2
```

## Root Cause

`repair` does this:

1. Runs `$(VENV_BIN)/python3 -m pip install --force-reinstall ...`
2. Removes `$(STAMP)/setup-dev.done`
3. Re-runs `$(STAMP)/setup-dev.done`

But `setup-dev.done` currently does:

```make
python3 -m venv $(VENV_DIR)
$(VENV_BIN)/pip install --quiet --upgrade pip
$(VENV_BIN)/pip install --quiet -r $(MAIN_ROOT)/requirements.txt
```

If the workspace path changed, the recreated `.venv/bin/pip` script can still point at the old interpreter path, so the target immediately fails when it tries to execute `$(VENV_BIN)/pip`.

## Manual Workaround That Succeeded

This repaired the environment successfully:

```bash
.venv/bin/python3 -m ensurepip --upgrade
.venv/bin/python3 -m pip install --upgrade pip
.venv/bin/python3 -m pip install -r requirements.txt
.venv/bin/pre-commit install
.venv/bin/python3 -m pip install --force-reinstall pip
make validate
```

After that, `make validate` passed.

## Suggested Fix

Update the setup/repair flow to avoid executing venv `pip` wrapper scripts directly during repair/bootstrap. Prefer:

```make
$(VENV_BIN)/python3 -m pip install --quiet --upgrade pip
$(VENV_BIN)/python3 -m pip install --quiet -r $(MAIN_ROOT)/requirements.txt
```

That makes the flow resilient when `.venv/bin/pip` has a stale shebang.

## Scope

This looks like a workspace issue in the Makefile/dev-environment bootstrap flow, not a project repo issue.

---
**Authored-By**: `Codex`
**Model**: `GPT-5`

--- a/Makefile
+++ b/Makefile
@@ -140,8 +140,8 @@ $(STAMP):
 $(STAMP)/setup-dev.done: requirements.txt | $(STAMP)
 	@echo "--- Setting up dev tools ---"
 	python3 -m venv $(VENV_DIR)
-	$(VENV_BIN)/pip install --quiet --upgrade pip
-	$(VENV_BIN)/pip install --quiet -r $(MAIN_ROOT)/requirements.txt
+	$(VENV_BIN)/python3 -m pip install --quiet --upgrade pip
+	$(VENV_BIN)/python3 -m pip install --quiet -r $(MAIN_ROOT)/requirements.txt
 	$(PRE_COMMIT) install
 	touch $@
 

